File: VB/ConsoleApp/Shapes/Circle.vb
Imports ConsoleApp.Printers

Namespace Shapes

    Public Class Circle
        Inherits Shape
        Implements IPrintable

        Public Property Size As Integer = 0

        Public Sub New(x As Integer, y As Integer, size As Integer)
            Me.X = x
            Me.Y = y
            Me.Size = size
        End Sub

        Public Overrides Function ToString() As String Implements IPrintable.ToString
            Return String.Format("Circle ({0},{1}) size={2}", X, Y, Size)
        End Function

    End Class

End Namespace

File: CSharp/ConsoleApp/ConsoleApp/Concrete/Shapes/Circle.cs
using ConsoleApp.Abstractions;

namespace ConsoleApp.Concrete.Shapes
{
    public class Circle : Shape, IPrint
    {
        public int Size { get; set; }

        public Circle(int x = 1, int y = 1, int size = 300)
        {
            this.X = x;
            this.Y = y;
            this.Size = size;
        }

        public override string ToString()
        {
            return string.Format("Circle ({0},{1}) size={2}", X, Y, Size);
        }
    }
}
